src/compas_fab/robots/semantics.py

```diff
@@ -122,23 +184,78 @@ def from_xml(cls, xml, robot_model):
         )
 
     def get_end_effector_link_name(self, group=None):
+        """Get the :obj:`str` name of the last link (end effector link) in a planning group.
```

Comment:
```suggestion
        """Get the name of the last link (end effector link) in a planning group.
```
doesn't it read weird to see ```Get the `str` name of the ...```? A name is a string anyway

Reply:
I guess a name is a str... Okay.